src/main/java/org/opensearch/flowframework/util/ParseUtils.java

```diff
@@ -78,4 +85,35 @@ public static Map<String, String> parseStringToStringMap(XContentParser parser)
         return map;
     }
 
+    /**
+     * Parse content parser to {@link java.time.Instant}.
+     *
+     * @param parser json based content parser
+     * @return instance of {@link java.time.Instant}
+     * @throws IOException IOException if content can't be parsed correctly
+     */
+    public static Instant toInstant(XContentParser parser) throws IOException {
```

Comment:
For consistency with the other methods:
```suggestion
    public static Instant parseInstant(XContentParser parser) throws IOException {
```

Reply:
fixed